editions/tw5.com/tiddlers/filters/allafter.tid

```diff
@@ -0,0 +1,17 @@
+caption: allafter
+created: 20151017145021839
+creator: matabele
```

Comment:
Hi @matabele we avoid the creator/modifier fields in tw5.com; the idea is that there's a single, unified editorial voice.